config/settings.yaml.test

```diff
@@ -1,5 +1,5 @@
 ---
 :unattended: true
 :require_ssl: false
-:fqdn: foreman.example.com
+:fqdn: foreman.some.host.fqdn
```

Comment:
why is this change needed? I don't see it reflected in the tests

also, please use `example.test` or `example.invalid`, not a randomly invented `.fqdn` tld ;)

Reply:
Well... this is the value that the default setting got from fixtures and until this point, we did not reflect the `fqdn` value, because test were overriding it with fixture. So I chose to set that value to what tests expect instead of changing all the tests, but I can change the tests instead :)

Reply:
Then IMHO the fixtures should be changed. https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600 should be leading here. Historically people have not respected it, but I think we should.